packages/react-ui-core/src/Gmap/withGoogleScript.js

```diff
@@ -65,8 +64,14 @@ export default function(BaseComponent) {
 
     loadScript() {
       const { apiKey, libraries, version } = this.props
-      const url = `${API_BASE_URL}?key=${apiKey}&version=${version}&libraries=${libraries.join()}`
-      scriptjs(url, this.scriptLoaded)
+      const script = document.createElement('script')
+
+      window.google_map_initialize = this.scriptLoaded
+
+      script.async = true
+      script.defer = true
+      script.src = `${API_BASE_URL}?key=${apiKey}&version=${version}&libraries=${libraries.join()}&callback=google_map_initialize`
```

Comment:
Should we add a script.onerror in case the script doesn't load correctly? We could add a window flag if the map errors out we could check, potentially, not sure where else we would put it - since, google_map_initialize would never run, we would have no way to handle that. I guess we wouldn't have a plan of what to do with it if it did fail, but we would have the option to do so (would be something to bring up with team prod)

Reply:
Done!